templates/apollo-starter-ts/src/config/config.ts

```diff
@@ -12,8 +12,8 @@ class Config {
     this.port = process.env.PORT || 4000
     this.db = {
       database: process.env.DB_NAME || 'users',
-      user: process.env.DB_USERNAME || 'postgres',
-      password: process.env.DB_PASSWORD || 'password',
+      user: process.env.DB_USERNAME || 'postgresql',
```

Comment:
It looks strange and it is very postgres specific. Do we still do pluggable db idea?

Reply:
Because the files for db connection are specific, maybe db specific templates is a way to go
like this can be `apollo-postgres-typescript`

Reply:
hmm.. We agreed that we will have separate abstraction for databases added to generator. We have this in form of docker-compose file when we choose this in CLI. This is tech debt - let's create separate issue to address it.

Reply:
TL:DR let's not do it now today - but this is code smell that needs to be addressed before 1.0 :)